cropper/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,14 @@
-<manifest
-    package="com.theartofdev.edmodo.cropper">
-
+<manifest package="com.theartofdev.edmodo.cropper"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <application>
+        <provider
+            android:name=".ImageCropperFileProvider"
+            android:authorities="${applicationId}.fileprovider"
```

Comment:
`.fileprovider` is easily the default suffix for anyone starting using file providers.

Use a library-specific suffix, like `${applicationId}.com.theartofdev.edmodo.cropper.fileprovider`.